packages/react-toolbar/src/components/ToolbarButton/useToolbarButton.ts

```diff
@@ -13,19 +15,8 @@ import type { ToolbarButtonProps, ToolbarButtonState } from './ToolbarButton.typ
  */
 export const useToolbarButton_unstable = (
   props: ToolbarButtonProps,
-  ref: React.Ref<HTMLElement>,
+  ref: React.Ref<HTMLButtonElement | HTMLAnchorElement>,
 ): ToolbarButtonState => {
-  return {
-    // TODO add appropriate props/defaults
-    components: {
-      // TODO add each slot's element type or component
-      root: 'div',
-    },
-    // TODO add appropriate slots, for example:
-    // mySlot: resolveShorthand(props.mySlot),
-    root: getNativeElementProps('div', {
-      ref,
-      ...props,
-    }),
-  };
+  const size = useContextSelector(ToolbarContext, ctx => ctx.size);
```

Comment:
you never use the `size` prop that you've declared in the types, should user be able to override the size on the component ?

Reply:
I think it should be possible. but not the default behavior?!

Reply:
Do you think we shouldn't allow it?